Week2/5.Round-Compare.js

```diff
@@ -0,0 +1,25 @@
+// 5.1
+let z = 7.25 ;
+
+// 5.2
+console.log(z) ;
+
+// 5.3
+let a = Math.round(z) ;
+
+// 5.4
+console.log(a) ;
+
+// 5.5
+if (a > z) {
+    theHighestOne = a ;
+}
+    else {
+    theHighestOne = z ;    
+}
```

Comment:
A goodgood, there is an easier solution though, with [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)